android/src/main/java/com/smileidentity/react/Mapper.kt

```diff
@@ -153,23 +154,44 @@ fun ReadableMap.toEnhancedKycRequest(): EnhancedKycRequest {
     },
     consentInformation = getMapOrDefault("consentInformation", null)?.toConsentInfo() ?: run {
       ConsentInformation(
-        consentGrantedDate = getCurrentIsoTimestamp(),
-        personalDetailsConsentGranted = false,
-        contactInfoConsentGranted = false,
-        documentInfoConsentGranted = false
+        consented = ConsentedInformation(
+          consentGrantedDate = getCurrentIsoTimestamp(),
+          personalDetails = false,
+          contactInformation = false,
+          documentInformation = false
+        )
       )
     },
   )
 }
 
 fun ReadableMap.toConsentInfo(): ConsentInformation {
+  val consentGrantedDate = getStringOrDefault("consentGrantedDate", null) ?: getCurrentIsoTimestamp()
+
+  // Try the new property names first, fall back to old property names if new ones aren't present
+  val personalDetailsConsentGranted = if (hasKey("personalDetailsConsentGranted")) {
+    getBoolOrDefault("personalDetailsConsentGranted", false)
+  } else {
+    getBoolOrDefault("personalDetails", false)
+  }
```

Comment:
```suggestion
  val personalDetailsConsentGranted = getBoolOrDefault("personalDetails", getBoolOrDefault("personalDetailsConsentGranted", false)
```
Not sure it that's a better representation?